Dockerfiles/agent/entrypoint/51-docker.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# We enable docker if either:
+#   - we detect the DOCKER_HOST envvar, overriding the default socket location
+#     (in that case, we trust the user wants docker integration and don't check existence)
+#   - we find the docker socket at it's default location
+
+if [[ -z "${DOCKER_HOST}" && ! -e /var/run/docker.sock ]]; then
+    exit 0
+fi
+
+
+cd /etc/datadog-agent/
```

Comment:
I would prefer avoid `cd` in this kind of scripts and work with absolute paths even if it's more verbose it's lets prone to errors, WDYT?